addons/web/controllers/main.py

```diff
@@ -1585,6 +1586,49 @@ def get_app_url(self, req, app):
         sakey = Session().save_session_action(action)
         return werkzeug.utils.redirect('/web#sa={0}'.format(sakey))
 
+class AttachmentController(http.Controller):
+
+    @http.route('/attachment/download_pdfs', type='http', auth='user')
+    def merge_and_download_pdfs(self, attachment_ids, **data):
+        def _close_streams(streams):
+            for stream in streams:
+                try:
+                    stream.close()
+                except Exception:
+                    pass
+
+        ids = [int(id) for id in attachment_ids.split(',')]
+        original_attachments = request.env['ir.attachment'].browse(ids)
+
+        streams = []
+        readers = []
+        for attachment in original_attachments:
+            if (attachment.name.endswith(".pdf") and attachment.datas):
```

Comment:
why only pdf? It shouldn't be the concern of this method

Reply:
I mean: I could have one bill as a PDF and another one as a JPG (created through the OCR or whatever)